src/scheduler/scheduler.py

```diff
@@ -166,16 +170,31 @@ def _finalize_task_result(self, started, finished, status, detail):
                 logger.debug("Trying callback")
                 context = {"request": self.entry.request}
                 result_json = TaskResultSerializer(tr, context=context).data
-                token = self.entry.owner.auth_token
-                headers = {"Authorization": "Token " + str(token)}
-                verify_ssl = settings.CALLBACK_SSL_VERIFICATION
-                requests_futures_session.post(
-                    self.entry.callback_url,
-                    json=result_json,
-                    background_callback=self._callback_response_handler,
-                    headers=headers,
-                    verify=verify_ssl,
-                )
+                if settings.CALLBACK_SSL_VERIFICATION:
+                    verify_ssl = settings.OAUTH_PATH_TO_VERIFY_CERT
```

Comment:
Need a way to allow a user to use CALLBACK_SSL_VERIFICATION without specifying verify cert. In that case, it should be set to True to allow standard SSL cert verification (using standard, common certificate authorities).